javascript-local-storage/data.js

```diff
@@ -0,0 +1,13 @@
+/* exported todos */
+
+var todos = [];
+
+function handleWindow(event) {
+  var todosJSON = JSON.stringify(todos);
+  localStorage.setItem('javascript-local-storage', todosJSON);
+}
+
+window.addEventListener('beforeunload', handleWindow);
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
make sure you wrap this line in a conditional statement to confirm that `previousTodosJson` is not null